package.json

```
@@ -1,6 +1,6 @@
 {
-    "name": "ionic2-calendar",
-    "version": "0.5.8",
+    "name": "ionic5-calendar",
```

```suggestion
    "name": "ionic2-calendar",
```

I don't think the package name should change for this minor update.